src/sentry/static/sentry/app/views/eventsV2/eventDetails/content.tsx

```diff
@@ -163,29 +158,10 @@ class EventDetailsContent extends AsyncComponent<Props, State> {
             <StyledButton size="small" onClick={this.toggleSidebar}>
               {isSidebarVisible ? 'Hide Details' : 'Show Details'}
             </StyledButton>
-            {isGroupedView && (
-              <Pagination
```

Comment:
@markstory would you be following up with a PR to remove the code paths on the django side that fetches the next/previous event links?

Reply:
Yes. That would be 🌈 🌈 as those queries are expensive.